config.json

```diff
@@ -1,6 +1,5 @@
 {
   "language": "kotlin",
-  "output": "./generated/",
```

Comment:
What did this do?

Reply:
just causes it to use the default directory `client` when generating locally. since `client` is already in .gitignore I figured that was griffin's intention.